tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -697,6 +697,7 @@ public virtual void DoSomething()
 
             public void DoSomethingNonVirtual()
             {
+                this.WasCalled = true;

```

Comment:
No, it's just so that the method can no longer be static, to avoid the warning. Which is exactly the kind of thing that makes me think re-enabling CA1822 isn't such a good idea, at least for test projects.